x/emissions/keeper/keeper.go

```diff
@@ -1044,24 +1070,27 @@ func (k *Keeper) AppendInference(
 }
 
 // Insert a complete set of inferences for a topic/block. Overwrites previous ones.
```

Comment:
This comment is now outdated, right? Belongs to the next one below "insertActiveInferences" maybe?